packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -37,7 +37,8 @@ export const ListBoxExample = (args) => (
 ListBoxExample.story = {
   args: {
     selectionMode: 'none',
-    selectionBehavior: 'toggle'
+    selectionBehavior: 'toggle',
+    shouldFocusOnHover: false
```

Comment:
did we want this to be an argTypes? or is it always supposed to be false?

Reply:
storybook infers that this is a toggle control by just setting it as an boolean here

Reply:
amaze